### ISSUE TITLE
Significant event markers

### ISSUE THREAD

**Issue by philipa** [meri-imperiumi/signalk-logbook#38]
It would be nice to indicate that a log event marked a significant boundary. E.g. the start or end of a voyage. This could be indicated by a strong ruled line across the log.

**Comment by bergie** [comment]
Yes, and I guess in the map view we could use custom markers (or just colours) for those.

In my previous stab at this, I started a new table for each trip. But it was quite difficult to get those to align.

**Comment by philipa** [comment]
Maybe give the row a class based on the event category, then use CSS.
